tests/PHPUnit/System/expected/test_CustomEvents_Events.getAction_flat__API.getProcessedReport_day.xml

```diff
@@ -235,7 +231,7 @@
 		<result prettyDate="Saturday 9 January 2010" />
 	</reportMetadata>
 	<reportTotal>
-		<nb_visits>38</nb_visits>
-		<nb_uniq_visitors>38</nb_uniq_visitors>
+		<nb_visits>42</nb_visits>
```

Comment:
This is a fix as ReportsTotalsCalculator does no longer operate on the flattened table but only on the first level of the report which is more accurate